multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/util/ApplicationConfiguration.java

```diff
@@ -218,13 +225,24 @@ public class ApplicationConfiguration {
     private Integer threadsForFileStorageUpload;
     private Boolean isHealthCheckEnabled;
 
-    public ApplicationConfiguration() {
-        this(new Environment());
+    private final ApplicationConfigurationAuditLog applicationConfigurationAuditLog;
+
+    public ApplicationConfiguration(ApplicationConfigurationAuditLog applicationConfigurationAuditLog) {
+        this(new Environment(), applicationConfigurationAuditLog);
     }
 
     @Inject
-    public ApplicationConfiguration(Environment environment) {
+    public ApplicationConfiguration(Environment environment, ApplicationConfigurationAuditLog applicationConfigurationAuditLog) {
         this.environment = environment;
+        this.applicationConfigurationAuditLog = applicationConfigurationAuditLog;
+    }
+
+    private void auditLog(String envVariableName, String space) {
```

Comment:
auditlog is a noun in the general context, the code would be more readable if the method was 'logInAuditlog' or something similar